src/Components/Logout/LogoutS.scss

```diff
@@ -0,0 +1,22 @@
+@import "../../Sass/main.scss";
+
+.Logout {
+    display: flex;
+    flex-direction: row-reverse;
+    margin-bottom: 2rem;
+
+    .logout-btn {
+        width: 10rem;
+        height: 3rem;
+
+        &:hover {
+            transform: translateY(-2px) scale(1.02);
+            box-shadow: 0 1rem 2rem rgba($color-black, 0.4);
+        }
+
+        &:active {
+            transform: translateY(0) scale(1);
+            box-shadow: 0 1rem 2rem rgba($color-black, 0);
+        }
+    }
```

Comment:
should we create a variable for the on-hover & active state for this basic effect?